cotel/src/main/scala/day2/SpreadsheetChecksum.scala

```diff
@@ -0,0 +1,35 @@
+package day2
+
+
+object SpreadsheetChecksum {
+  type Row = List[Int]
+  type SpreadSheet = List[Row]
+
+  def Row(xs: Int*) = List(xs: _*)
+
+  def SpreadSheet(rows: Row*) = List(rows: _*)
+
+  def checksum(spreadSheet: SpreadSheet): Int = {
+    spreadSheet
+      .map { list => (list.max, list.min) }
```

Comment:
I think list.max and list.min calling two functions?

or is filled when are adding the items?

Reply:
In Scala, functions with no parameters can be called without `()`. So I'm calling `list.max()` and `list.min()` really.